app/models/manageiq/providers/redhat/infra_manager/provision/disk.rb

```diff
@@ -0,0 +1,78 @@
+module ManageIQ::Providers::Redhat::InfraManager::Provision::Disk
+  def configure_disks
+    configure_dialog_disks
+    requested_disks = options[:disks_add]
+    if requested_disks.nil?
+      _log.info "Disks settings will be inherited from the template."
+      signal :customize_guest
+    else
+      add_disks(requested_disks)
+      signal :poll_add_disks_complete
```

Comment:
@gmcculloug not sure, but shouldnt the signals be consistent across providers? Or somehow defined at a base class?